lib/rubber/cloud/digital_ocean.rb

```diff
@@ -115,6 +137,25 @@ def describe_instances(instance_id=nil)
       def active_state
         'active'
       end
+
+      def destroy_instance(instance_id)
+        # The Digital Ocean API will return a 422 if we attempt to destroy an
+        # instance that's in the middle of booting up, so wait until it's
+        # in a non-"new" state
+        print "Waiting for non-new instance state"
```

Comment:
Okay. I think the biggest difference is cap logging will prepend with the host name or IP so if you're starting multiple instances, you'll be able to keep track of them. But I'd have to check that to be sure.